attributes/default.rb

```diff
@@ -79,6 +79,7 @@
 default['os-hardening']['auth']['pam']['tally2']['template_cookbook']        = 'os-hardening'
 default['os-hardening']['auth']['pam']['passwdqc']['template_cookbook']      = 'os-hardening'
 default['os-hardening']['auth']['pam']['system-auth']['template_cookbook']   = 'os-hardening'
+default['os-hardening']['auth']['pam']['pam_systemd']['enable']       = node['os-hardening']['desktop']['enable'] && node['platform'] == 'fedora'
```

Comment:
@avanier thanks for this PR! It looks like `pam_systemd`is related to all systemd based distros, however we [manage](https://github.com/dev-sec/chef-os-hardening/blob/aa9202b28497c26dbb68316e137ed7d227a8af70/recipes/pam.rb#L124-L131) this particular pam configuration only for rhel/fedora. My suggestion is to remove `&& node['platform'] == 'fedora'` here because of that